exercices/part2/practice1.py

```diff
@@ -0,0 +1,8 @@
+
+
+if __name__ == "__main__":
+    '''Create a syntax error of your own by entering your favorite ice cream flavor'''
+    '''strawberry'''
+
+    '''Enter a statement that prints the name of your favorite ice cream.'''
+    print("strawberry")
```

Comment:
New line needed

Reply:
ok, i will do that, thank you for the hint :)